### ISSUE TITLE
When a coach user copies a Quiz from one class to other and assigns it to individuals, it breaks the second Class's Plan > Quizzes page

### ISSUE THREAD

**Issue by JatinSharma12** [learningequality/kolibri#7608]
### Observed behavior
When a coach user copies a Quiz from one class to other, it gets copied but when we try to open the copied quiz from the other class, the page breaks and app stops [responding.
](https://github.com/learningequality/kolibri/blob/release-v0.14.x/integration_testing/features/coach/coach-copy-quiz.feature)
…

### Expected behavior
Application should not stop and user should be able to review the copied Quiz.

…

### User-facing consequences
User might lose lot of data/work as app stops working
…

### Errors and logs

Relevant logs from:
[kolibri-app.txt](https://github.com/learningequality/kolibri/files/5429351/kolibri-app.txt)

```

```

…

### Steps to reproduce
1. Login as Coach.
2. Copy Quiz form one class to another.
3. Try to open copied Quiz from the other class.
4. Notice that user is not able to access that page and application stops responding.

PFA the video for reference.
[CopyQuiz_issue.zip](https://github.com/learningequality/kolibri/files/5429370/CopyQuiz_issue.zip)

…

### Context
Tell us about your environment, including:
 * Kolibri version 0.14.3
 * Operating system Windows 10

…

**Comment by jonboiser** [comment]
I was able to reproduce this with 0.14.4-weekly5. A critical detail in the video is that the Quiz needs to be copied and then assigned to Individual Learners. If I pick "Entire class" I don't get this error.

When I do pick individual learners, I'm not able to see the entire Coach > Plan >Quizzes page, and I see these errors in the console

```
kolibri.core.default_frontend-0.14.4.dev0.js:145572 [Vue warn]: Error in render: "TypeError: Cannot read property 'member_ids' of undefined"

found in

---> <QuizzesBlock> at kolibri/plugins/coach/assets/src/views/home/HomePage/QuizzesBlock.vue
       <KFixedGridItem> at node_modules/kolibri-design-system/lib/grids/KFixedGridItem.vue
         <KGridItem> at node_modules/kolibri-design-system/lib/grids/KGridItem.vue
           <KFixedGrid> at node_modules/kolibri-design-system/lib/grids/KFixedGrid.vue
             <KGrid> at node_modules/kolibri-design-system/lib/grids/KGrid.vue
               <KFixedGridItem> at node_modules/kolibri-design-system/lib/grids/KFixedGridItem.vue
                 <KGridItem> at node_modules/kolibri-design-system/lib/grids/KGridItem.vue
                   <KFixedGrid> at node_modules/kolibri-design-system/lib/grids/KFixedGrid.vue
                     <KGrid> at node_modules/kolibri-design-system/lib/grids/KGrid.vue
                       <CoreBase> at kolibri/core/assets/src/views/CoreBase/index.vue
                         <HomePage> at kolibri/plugins/coach/assets/src/views/home/HomePage/index.vue
                           <Root>
warn @ kolibri.core.default_frontend-0.14.4.dev0.js:145572
kolibri.core.default_frontend-0.14.4.dev0.js:146839 TypeError: Cannot read property 'member_ids' of undefined
    at kolibri.plugins.coach.app-0.14.4.dev0.js:2103
    at arrayMap (kolibri.plugins.coach.app-0.14.4.dev0.js:47554)
    at map (kolibri.plugins.coach.app-0.14.4.dev0.js:58207)
    at VueComponent.<anonymous> (kolibri.plugins.coach.app-0.14.4.dev0.js:2103)
    at VueComponent.<anonymous> (kolibri.plugins.coach.app-0.14.4.dev0.js:2147)
    at kolibri.plugins.coach.app-0.14.4.dev0.js:29083
    at Array.map (<anonymous>)
    at VueComponent.table (kolibri.plugins.coach.app-0.14.4.dev0.js:29082)
    at Watcher.get (kolibri.core.default_frontend-0.14.4.dev0.js:149425)
    at Watcher.evaluate (kolibri.core.default_frontend-0.14.4.dev0.js:149530)
logError @ kolibri.core.default_frontend-0.14.4.dev0.js:146839
kolibri.core.default_frontend-0.14.4.dev0.js:145572 [Vue warn]: Error in render: "TypeError: Cannot read property 'member_ids' of undefined"

found in

---> <CoachExamsPage> at kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue
       <Root>
warn @ kolibri.core.default_frontend-0.14.4.dev0.js:145572
kolibri.core.default_frontend-0.14.4.dev0.js:146839 TypeError: Cannot read property 'member_ids' of undefined
    at kolibri.plugins.coach.app-0.14.4.dev0.js:2103
    at arrayMap (kolibri.plugins.coach.app-0.14.4.dev0.js:47554)
    at map (kolibri.plugins.coach.app-0.14.4.dev0.js:58207)
    at Proxy.<anonymous> (kolibri.plugins.coach.app-0.14.4.dev0.js:2103)
    at Proxy.<anonymous> (kolibri.plugins.coach.app-0.14.4.dev0.js:2125)
    at kolibri.plugins.coach.app-0.14.4.dev0.js:84742
    at Proxy.renderList (kolibri.core.default_frontend-0.14.4.dev0.js:147579)
    at Proxy.render (kolibri.plugins.coach.app-0.14.4.dev0.js:84713)
    at VueComponent.Vue._render (kolibri.core.default_frontend-0.14.4.dev0.js:148494)
    at VueComponent.updateComponent (kolibri.core.default_frontend-0.14.4.dev0.js:149012)
```

**Comment by jonboiser** [comment]
I'm fixing this issue as part of the work to fix #7580. The root cause is that when an exam is copied, it actually sends the classroom ID of the original class as one of the assignments for the copy. So when you're viewing a different classroom, that original class's ID doesn't refer to a group or anything in the new classroom, hence the original classroom ID as a key for the new classroom's group map returns `undefined` and you get `Cannot read property 'member_ids' of undefined`
